cmd/dtsmble.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2023 Interlynk.io
```

Comment:
Here is what i would like, sbomasm has two commands 
- assemble
- edit 

we want to provide both these functionalities to dt. I would classify DT as an input-output source hence we should augment the current commands to take in DT related information. In sbomqs i made the mistake of making it a separate command. 

e.g 

This command indicates that dt is in input source from where to get the input files from project-ids.  We should allow mixing of project-ids and file-names.  If file-names are provided that means they are local files.  The output assembled file is now stored in the file-system.
```
sbomasm assemble -n "mega cdx app" -v "1.0.0" -t "application" --dt-hosturl"http://localhost:8080" --dt-api-key "asadasdas" -o final-product.cdx.json sbom1.json <project-id1> <project-id2> <project-id3> .....
```

The following command is the same as above except the output is pushed back to DT 
```
sbomasm assemble -n "mega cdx app" -v "1.0.0" -t "application" --dt-hosturl"http://localhost:8080" --dt-api-key "asadasdas" --dt-output <project-id-5> sbom1.json <project-id1> <project-id2> <project-id3> .....
```

Now this can also be replicated to `edit` command at a later date.

Reply:
added those changes